From 777f02a71e3c947f93324b70da5a53ecbcae967c Mon Sep 17 00:00:00 2001
From: Harini Katakam <harini.katakam@xilinx.com>
Date: Thu, 27 Mar 2014 15:10:25 +0530
Subject: [PATCH 483/509] GPIO: zynq: Remove dev_info print in probe

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 1d6ec13e113feaeef5c25b0b2b12eb25ee654ebb

Remove dev_info print in probe as per mainline comments.

Signed-off-by: Harini Katakam <harinik@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/gpio/gpio-zynq.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/gpio/gpio-zynq.c b/drivers/gpio/gpio-zynq.c
index 50201ba..0a265cb 100644
--- a/drivers/gpio/gpio-zynq.c
+++ b/drivers/gpio/gpio-zynq.c
@@ -636,9 +636,6 @@ static int zynq_gpio_probe(struct platform_device *pdev)
 	if (ret < 0)
 		return ret;
 
-	dev_info(&pdev->dev, "gpio at 0x%08lx mapped to 0x%08lx\n",
-		 (unsigned long)res->start, (unsigned long)gpio->base_addr);
-
 	/* Enable GPIO clock */
 	gpio->clk = devm_clk_get(&pdev->dev, NULL);
 	if (IS_ERR(gpio->clk)) {
-- 
1.7.5.4

